diesel/src/expression/expression_methods/text_expression_methods.rs

```diff
@@ -1,8 +1,39 @@
 use expression::{Expression, AsExpression};
-use expression::predicates::{Like, NotLike};
+use expression::predicates::{Concat, Like, NotLike};
 use types::Text;
 
 pub trait TextExpressionMethods: Expression<SqlType=Text> + Sized {
+    /// Concatenates two strings using the `||` operator.
+    ///
+    /// # Example
```

Comment:
Convention in std is to always write "Examples" (plural). (Not important and we should probably define some doc comment rules before changing stuff like this. Also, I'll write a Rust RFC on that… soonish)